cypress/e2e/models/migration/migration-waves/migration-wave.ts

```diff
@@ -136,6 +137,7 @@ export class MigrationWave {
         MigrationWave.open();
         this.expandActionsMenu();
         cy.contains(manageApplications).click();
+        MigrationWave.selectItemsPerPageManageApplication(100);
```

Comment:
Use selectItemsPerPage(itemsPerPage); instead 

Example 
https://github.com/konveyor/tackle-ui-tests/blob/main/cypress/e2e/models/migration/applicationinventory/assessment.ts#L76

Reply:
Thanks for the advice. Done